cost/aws/superseded_ebs_volumes/README.md

```diff
@@ -0,0 +1,102 @@
+# AWS Rightsize EBS Volumes
+
+## What It Does
+
+This policy checks all GP2 volumes on an AWS Account to see if the GP3 equivalent would be less expensive. An incident is raised with all volumes that would be less expensive if upgraded from GP2 to GP3.
+
+## Functional Details
+
+- The policy leverages the AWS API to retrieve a list of all volumes in an AWS Account
+- The policy identifies all GP2 volumes and uses the AWS Pricing API to retrieve the current cost, and the cost of the volume if it were a GP3 volume type.
+- If there is a cost savings associated with moving the volume type from GP2 to GP3, this will provide a recommendation.
+
+### Policy savings details
+
+The policy includes the estimated savings. The estimated savings is recognized if the volume is upgraded from GP2 to GP3. The AWS Pricing API is used to retrieve and calculate the estimated savings which is the expected GP3 cost subtracted from the estimated current GP2 cost of the volume. The incident message detail includes the sum of each resource *Estimated Monthly Saving* as *Potential Monthly Savings*.
+
+If the Flexera organization is configured to use a currency other than USD, the savings values will be converted from USD using the exchange rate at the time that the policy executes.
+
+### Policy Savings Details
```

Comment:
This header appears in the README twice.

Reply:
I don't know why this was marked as "resolved". The issue is still present. Look at lines 13 and 19. There should only be one "Policy Savings Details" section, not two.

Reply:
Removed in b469698 .